src/warnet/warnet.py

```diff
@@ -90,6 +90,7 @@ def from_graph(cls, graph):
     def from_network(
         cls, config_dir: Path = Path(), network: str = "warnet", tanks=True
     ):
+        config_dir = gen_config_dir(network)
```

Comment:
this would override the passed-in argument. is it still a required param to from_network?